data/json/npcs/exodii/exodii_merchant_talk.json

```diff
@@ -567,7 +565,7 @@
     "id": "TALK_EXODII_MERCHANT_Experiment_Prototype_Cyborg_Rescue",
     "type": "talk_topic",
     "dynamic_line": {
-      "//~": "For this one I can pull out their cbm and give any working to you, or I can just take out broken bits so that they will still be able to use the rest of stuff in them.  Alternatively why don't you just give them to me I surely can help them adjust to life as a cyborg.'",
+      "//~": "For this one, I can pull out their CBM and give any working to you, or I can just take out broken bits so that they will still be able to use the rest of stuff in them.  Alternatively why don't you just give them to me I surely can help them adjust to life as a cyborg.",
```

Comment:
```suggestion
      "//~": "For this one, I can pull out their CBM and give any working to you, or I can just take out broken bits so that they will still be able to use the rest of stuff in them.  Alternatively why don't you just give them to me?  I surely can help them adjust to life as a cyborg.",
```